Source/Parallelization/WarpXComm_K.H

```diff
@@ -44,18 +44,18 @@ void warpx_interp (int j, int k, int l,
 
     // Refinement ratio
     const int rj = rr[0];
-    const int rk = (AMREX_SPACEDIM == 1) ? 1 : rr[1];
-    const int rl = (AMREX_SPACEDIM <= 2) ? 1 : rr[2];
+    const int rk = (AMREX_SPACEDIM > 1) ? rr[1] : 1;
+    const int rl = (AMREX_SPACEDIM > 2) ? rr[2] : 1;
 
     // Staggering (0: cell-centered; 1: nodal)
     const int sj = arr_stag[0];
-    const int sk = (AMREX_SPACEDIM == 1) ? 0 : arr_stag[1];
-    const int sl = (AMREX_SPACEDIM <= 2) ? 0 : arr_stag[2];
+    const int sk = (AMREX_SPACEDIM > 1) ? arr_stag[1] : 1;
+    const int sl = (AMREX_SPACEDIM > 2) ? arr_stag[2] : 1;
```

Comment:
This seems to change the behavior of the code.
For instance, if  `AMREX_SPACEDIM` is 1, in the previous version `sk` would be 0, while now it would be `1`. Is this intended?

Reply:
That was intended. If you follow through the original code, for the unused dimensions, the value of `sk` doesn't matter, it produces the same result whether it is 0 or 1. I thought it made it clearer what is happening to set `sk` to 1 since in some sense the data is nodal in that direction.